src/components/OpenTofuNotice.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import Admonition from "@theme/Admonition"
+
+export const OpenTofuNotice = () => {
+    return (
+        <div>
```

Comment:
Nit: I don't think the outer `<div>` wrapper is necessary.

Reply:
removed in [724e207](https://github.com/gruntwork-io/docs/pull/1156/commits/724e2076d68e5cd177c841a1ebb7e8238d43ca43)